pkg/pool-stable/contracts/StablePool.sol

```diff
@@ -256,10 +256,7 @@ contract StablePool is BaseGeneralPool, LegacyBaseMinimalSwapInfoPool, IRateProv
         address,
         uint256[] memory scalingFactors,
         bytes memory userData
-    ) internal virtual override whenNotPaused returns (uint256, uint256[] memory) {
-        // It would be strange for the Pool to be paused before it is initialized, but for consistency we prevent
-        // initialization in this case.
```

Comment:
This comment was correct, it should not be removed.

Reply:
It got moved up to LegacyBasePool, where it's disabled, not removed.

Reply:
Moving the check from StablePool up to LegacyBasePool also affects Linear & PhantomStable (had to skip tests that would fail). Should we put it back? Maybe it should only be moved after all the other pools support recovery mode.